var/spack/repos/builtin/packages/blender/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+
+from spack import *
+
+import os
+import shutil
+
+class Blender(Package):
+    """Blender is the free and open source 3D creation suite."""
+
+    homepage = "https://www.blender.org"
+
+    version('2.81a-217', sha256='bb6e03ef79d2d7273336f8cfcd5a3b3f')
+
+    conflicts("python")
+
+    def install(self, spec, prefix):
+	for file in os.listdir(self.stage.source_path):
+            src = os.path.join(self.stage.source_path, file)
+            dst = os.path.join(prefix, file)
+            if os.path.isdir(src):
```

Comment:
This seems to be binary release. In that case, can we add conflict for non-supported platforms?

Reply:
Yes, it is a binary release. 
The only conflict specyfing targets I found it the repo are `ppc64`  and `ppc64le`. Is it enough?

Reply:
this is only for Linux so we need to add conflict for OS X and PowerPC